backend/src/StamAcasa.IdentityServer/Views/Shared/_Layout.cshtml

```diff
@@ -12,10 +12,11 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>IdentityServer4</title>
+    <title>Stăm acasă</title>
     <link rel="icon" type="image/x-icon" href="~/favicon.ico" />
```

Comment:
Can you please make sure that favicons are also set as [in here](https://github.com/code4romania/stam-acasa/blob/develop/frontend/public/index.html#L5-L17). Please remove `%PUBLIC_URL%` part in your case.

Reply:
Updated!